### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by joegreen2**
Creating documentation for repeated release of code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+
+#### keep it clean
+
+* Select a style guide a setup your IDE to help make it easier to follow.
+* [AirBnB style guide](https://github.com/airbnb/javascript) is suggested.
+* As good practice and especially when working with multiple developers, it is highly suggested to use a CI such as [CircleCI](https://circleci.com/), this is optional.
+
+#### bringin' it in...
+
+* Once tests have past and you are ready to merged your code from it's current branch such as: dev -> master, you are now ready for a [pull request](https://help.github.com/articles/creating-a-pull-request/).
```
Comment (MROwens) [inline README.md]: I would use passed instead of past here.

File: README.md [hunk 2]
```diff
@@ -1 +1,53 @@
...
+
+#### says who!?!
+
+* Depending on your project and environment you may have a "release manager" (the person that has to approve the code prior to sending it live).
+* Once your code is approved go ahead and send it down the pipe!
+* If you're not with current technology and like failing at life, go ahead a open your favorite FTP software and drop the files in the pretty window and that's it.
+
+#### It's alive!!!
+
+* Good job Frankenstein, now keep doing this for the rest of your life, and remember good documentation make a huge difference!
```
Comment (MROwens) [inline README.md]: "..good documentation _makes_ a huge difference."

File: README.md (hunk 1 above)
Comment (MROwens) [reply README.md]: merge instead of merged here as well

**Comment by Rhuansantos** [comment]
@joegreen2 Awesome job on your documentation 👍  LGTM! 🍻

**Comment by joegreen2** [comment]
Thanks for the support everyone! :beers:

**Pull request merged**